pkg/test/framework/components/echo/common/deployment/echos.go

```diff
@@ -283,7 +283,7 @@ func (c *Config) DefaultEchoConfigs(t resource.Context) []echo.Config {
 			Ports:           ports.All(),
 			Subsets:         []echo.SubsetConfig{{}},
 			IncludeExtAuthz: c.IncludeExtAuthz,
-			IPFamilies:      "IPv6, IPv4",
+			IPFamilies:      "IPv4",
 			IPFamilyPolicy:  string(corev1.IPFamilyPolicyRequireDualStack),
```

Comment:
won't this fail? Require but its not dual stack

Reply:
> won't this fail? Require but its not dual stack

Sorry this was a last minute edit after testing locally. Let me fix

Reply:
Fixed